.github/workflows/protocol.yml

```diff
@@ -34,30 +34,22 @@ jobs:
 
       - name: Unit tests
         working-directory: ./packages/protocol
-        run: pnpm clean && pnpm test
-
-      - name: Format solidity && update contract layout table
-        working-directory: ./packages/protocol
-        run: pnpm layout && forge fmt
+        run: pnpm clean && forge fmt && pnpm test:l1 && pnpm layout:l1 && pnpm test:l2 && pnpm layout:l2
 
       - name: Commit contract layout table
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
-          commit_message: "forge fmt & update contract layout table"
+          commit_message: "forge fmt & update contract layout tables"
 
       - name: Generate genesis
         working-directory: ./packages/protocol
         run: pnpm test:genesis
```

Comment:
Should it be `pnpm test:genesis:l2`? Always use this command to generate genesis on devnet

Reply:
It's OK to call it test:genesis